src/main/java/de/zalando/ep/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -467,7 +467,7 @@ private void timeout(String reason, ShutdownType shutdownType) {
         boolean shutDown = false;
 
         synchronized (this) {
-            if (this.testInformation != null) {
+            if (this.testInformation != null && this.testInformation.getTestStatus() == null) {
```

Comment:
This line was a bit strange... 
It had the consequence that all entries in dashboard's executedTestsInformation list had a TIMEOUT state when dumping  out or cleaning up.